workshops/personal_website/README.md

```diff
@@ -0,0 +1,239 @@
+
+```html
+<!DOCTYPE html>
+<html>
+<head>
+</head>
+<body>
+
+</body>
+</html>
+```
+
+### Step 4: Add Content To The Body
+
+- Let's type the following.
+- Mention copying and pasting vs re-typing
+
+```html
+<!DOCTYPE html>
+<html>
+<head>
+</head>
+<body>
+  I am Drake
+  All your hotlines belong to me.
+</body>
+</html>
+```
+
+### Step 5: Formatting the Heading
+
+```html
+<!DOCTYPE html>
+<html>
+<head>
+</head>
+<body>
+  <h1>I am Drake</h1>
+  All your hotlines belong to me.
+</body>
+</html>
+```
+
+### Step 6: Formatting the Paragraph
+
+```html
+<!DOCTYPE html>
+<html>
+<head>
+</head>
+<body>
+  <h1>I am Drake</h1>
+  <p>All your hotlines belong to me.</p>
+</body>
+</html>
+```
+
+### Step 7: Adding an Image
+
+```html
+<!DOCTYPE html>
+<html>
+<head>
+</head>
+<body>
+  <img src="https://media.giphy.com/media/vsyKKf1t22nmw/giphy.gif">
+  <h1>I am Drake</h1>
+  <p>All your hotlines belong to me.</p>
+</body>
+</html>
+```
+
+Our image is too big. CSS to the rescue.
+
+## Part III: CSS
+
+- What is CSS?
+- What are we going to use it for?
+  - Show before and after CSS
+
+### Creating the CSS file
+
+### Linking the CSS in the HTML
+
+```html
+<!DOCTYPE html>
+<html>
+<head>
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <img src="https://media.giphy.com/media/vsyKKf1t22nmw/giphy.gif">
+  <h1>I am Drake</h1>
+  <p>All your hotlines belong to me.</p>
+</body>
+</html>
+```
+
+### Resizing the Image
+
+```css
+img {
+    width: 200px;
+}
+```
+
+### Centering the Body
+
+```css
+img {
+    width: 200px;
+}
+
+body {
+    text-align: center;
+}
+```
+
+### Setting the Background Color
+
+```css
+img {
+    width: 200px;
+}
+
+body {
+    text-align: center;
+    background-color: #34495e;
+}
+```
+
+Use http://colorhunt.co to choose your favorite colors.
+
+<!-- I specifically like colorhunt.co because it only displays the hex colors and the user cannot copy and paste the color values so must retype them-->
+
+### Setting the Text Color
+
+```css
+img {
+    width: 200px;
+}
+
+body {
+    text-align: center;
+    background-color: #34495e;
+    color: #ffffff;
+}
+```
+
+### Changing the Font
+
+```css
+img {
+    width: 200px;
+}
+
+body {
+    text-align: center;
+    background-color: #34495e;
+    color: #ffffff;
+    font-family: Arial;
+}
+```
+
+## Part IV: Deployment
+
+### Getting the url of your website
+
+- Click on the green button with the arrow.
+- Copy and past the URL
+
+### Changing the Title
+
+```html
+<!DOCTYPE html>
+<html>
+<head>
+  <title>I am Drake</title>
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <img src="https://media.giphy.com/media/vsyKKf1t22nmw/giphy.gif">
+  <h1>I am Drake</h1>
+  <p>All your hotlines belong to me.</p>
+</body>
+</html>
+```
+
+## Part V: GitHub Pages
+
+- What is GitHub pages?
+- What are we going to use it for?
+- Let's create a repository named `username.github.io`
+- Let's run these git commands to create a repository
```

Comment:
Should this setup be done right when we create our GitHub account? If we create our Cloud9 workspace with the `account.github.io` repo as the "Clone URL", it sets up the git remote for us.